scrapi/base/__init__.py

```diff
@@ -211,12 +211,9 @@ def get_properties(self, result, property_list):
             prop.extend(result.xpath(
                 '//ns0:{}/node()'.format(item),
                 namespaces=self.NAMESPACES
-            ) or [''])
+            ))
 
-            if len(prop) > 1:
-                properties[item] = [self.copy_to_unicode(item) for item in prop]
-            else:
-                properties[item] = self.copy_to_unicode(prop[0])
+            properties[item] = [self.copy_to_unicode(element) for element in prop]
```

Comment:
This line has kinda bothered me for a while so:
```python
return {
    key: [
        copy_to_unicode(element)  # This should be a utility function rather than a class method
        for element in
        result.xpath('...'.format(key)) + # Consider moving these into private helpers
        result.xpath('...'.format(key))
    ]
    for key in properties_list
}
```